### ISSUE TITLE
Set metallic and roughness value directly

### ISSUE THREAD

**Issue by happydpc** [MeirBon/rfw-rs#10]
Can it use pbr pipeline by just set metallic and roughness values without any textures?

**Comment by MeirBon** [comment]
I thought I had added access functions to retrieve references to materials but I hadn't. I just added them. When you change a material it should automatically get updated in the renderer as well.

Btw, most of the stuff here is still a work in progress. I haven't had the time to finish most of the planned features for this project so there are still some rough edges. Thanks for checking it out though!

**Comment by happydpc** [comment]
yeah, I tried to modify the metallic and roughness value in physics example, seems that not much difference.
